src/tabs.js

```diff
@@ -63,4 +69,14 @@ export default class extends Controller {
   get anchor() {
     return (document.URL.split('#').length > 1) ? document.URL.split('#')[1] : null;
   }
+
+  /**
+   * Return iterable tab elements for either selectTarget or tabTarget cases
+   **/
+  get tabElements() {
+    if (this.hasTabTargets === undefined && this.hasSelectTarget) {
+      return Array.from(this.selectTarget.children);
+    }
```

Comment:
We only need to set tabElements to the children of the selectTarget IF tabTargets are not available. This is because tabTargets are utilized in part to set the css class `hidden` on their associated panels. When tabTargets are not available and selectTarget is, then we need to switch the tabElements to each child element of the selectTarget (e.g. the option elements within the select)